packages/logger/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@apppium/logger",
+  "version": "7.0.1",
+  "author": "https://github.com/appium",
+  "description": "A Universal Logger For The Appium Ecosystem",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/appium/appium.git",
+    "directory": "packages/logger"
+  },
+  "main": "build/index.js",
+  "types": "index.ts",
+  "files": [
+    "index.ts",
```

Comment:
other modules also include tsconfig.json, should we add it here?

Reply:
yea, added `"tsconfig.json"` and `"!build/tsconfig.tsbuildinfo"`